### ISSUE TITLE
generalization of `pragma(importPath)`

### ISSUE THREAD

**Issue by timotheecour** [marler8997/dlangfeatures#1]
@marler8997 
nim has very flexible pragmas for that:

https://nim-lang.org/docs/manual.html#implementation-specific-pragmas-compile-pragma
good for inspiration.  would be cool to have something extensible like that

**Comment by marler8997** [comment]
`pragma` is already extensible isn't it?

**Comment by timotheecour** [comment]
yes, what i meant is nim is an good example of a language that implemented the importpath u talked about, and IMO its good to see what else they used pragma for for inspiration.

=> https://github.com/timotheecour/timdstuff/wiki/scoped-compiler-options:-pragma(compilerOptions,-args...)
